src/neo/SmartContract/Helper.cs

```diff
@@ -35,7 +35,15 @@ public static bool Check(Script script, ContractAbi abi = null)
             Dictionary<int, Instruction> instructions = new Dictionary<int, Instruction>();
```

Comment:
Using https://github.com/neo-project/neo-vm/pull/391 we can avoid this dictionary